.github/workflows/all.yml

```diff
@@ -80,7 +80,8 @@ jobs:
       - name: Install and build all benchmarks and allocators
         # ff: error: use of undeclared identifier 'MAP_POPULATE'
         # gd: so many errors
-        run: ./build-bench-env.sh all no-lean no-gd no-ff
+        # lp: TODO(jvoisin)
```

Comment:
Is Mac still broken? Skimming the actions I couldn't see anything Mac specific that has failed?

Reply:
It's not broken, just disabled for now, since it was getting late here :)
I'll add osx support in a followup PR.